### ISSUE TITLE
import ib_insync fails on python 3.7.0

### ISSUE THREAD

**Issue by cyinwei** [erdewit/ib_insync#104]
Hello,

Complete newbie to `ib_insync` (looks super cool) here.  When running on `python 3.7.0`, I get an `invalid syntax`  error.  

```
# bottom of the stack trace
File "/Users/cyinwei/.venv/lib/python3.7/site-packages/ibapi-9.73.7-py3.7.egg/ibapi/client.py", line 60
    self.async = False
```

When I switched an ran on `python 3.6.6`, everything runs great.

This isn't a problem with `ib_insync` but with `ibapi` version 9.37 (the latest IB Api released in May 2018).  From a quick google search shows https://github.com/SublimeCodeIntel/CodeIntel/issues/5, where the `async` keyword is now reserved.

I'm not sure we can do anything on this side, just documenting in case someone else runs into the same problem (not sure an issue is the best documentation?).  Maybe in the readme it can warn not to run on `python 3.7.`

Here's the full stacktrace from the Jupyter notebook.
```
Traceback (most recent call last):

  File "/Users/cyinwei/.venv/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 3265, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-2-473401a31836>", line 1, in <module>
    import ib_insync

  File "venv/lib/python3.7/site-packages/ib_insync/__init__.py", line 24, in <module>
    from .ib import *

  File "/Users/cyinwei/.venv/lib/python3.7/site-packages/ib_insync/ib.py", line 11, in <module>
    from ib_insync.client import Client

  File "/Users/cyinwei/.venv/lib/python3.7/site-packages/ib_insync/client.py", line 10, in <module>
    from ibapi.client import EClient

  File "<frozen importlib._bootstrap>", line 983, in _find_and_load

  File "<frozen importlib._bootstrap>", line 963, in _find_and_load_unlocked

  File "<frozen importlib._bootstrap>", line 906, in _find_spec

  File "<frozen importlib._bootstrap_external>", line 1280, in find_spec

  File "<frozen importlib._bootstrap_external>", line 1254, in _get_spec

  File "<frozen importlib._bootstrap_external>", line 1235, in _legacy_get_spec

  File "<frozen importlib._bootstrap>", line 441, in spec_from_loader

  File "<frozen importlib._bootstrap_external>", line 594, in spec_from_file_location

  File "/Users/cyinwei/.venv/lib/python3.7/site-packages/ibapi-9.73.7-py3.7.egg/ibapi/client.py", line 60
    self.async = False
             ^
SyntaxError: invalid syntax
```

**Comment by JustinTArthur** [comment]
Unfortunately, this syntax problem is in IB's library. Until they've fixed it, I've taken to vendorizing the ibapi commenting that line out. At least in the released versions of their Python SDK, it's unused anyway.

**Comment by erdewit** [comment]
IB has fixed it in their repo but it's not known when that will be released. I've updated the readme page to tell users to comment out line 60 of ibapi/client.py if they're using Python 3.7.

**Comment by jarvisjohnson** [comment]
FYI this is fixed in the latest release of the IB API (9.74.01) - they've changed it to `self.asynchronous = False`
